src/state/slices/opportunitiesSlice/resolvers/thorchainsavers/utils.test.ts

```diff
@@ -0,0 +1,62 @@
+import type { Asset } from '@shapeshiftoss/asset-service'
+import { btcAssetId, osmosisAssetId } from '@shapeshiftoss/caip'
+import axios from 'axios'
+
+import { getThorchainSaversQuote } from './utils'
+
+jest.mock('axios')
+const mockAxios = axios as jest.Mocked<typeof axios>
+
+const btcQuote = {
+  expected_amount_out: '9997894',
+  fees: {
+    affiliate: '0',
+    asset: 'BTC/BTC',
+    outbound: '0',
+  },
+  inbound_address: 'bc1q0vphqevkhc33g94pl0ctnnp58v9mcuhp4e2hnm',
+  inbound_confirmation_blocks: 1,
+  memo: '+:BTC/BTC',
+  slippage_bps: 2,
+}
+
+const thorchainErrorResponse = {
+  error: 'Invalid pool',
+}
+
+describe('resolvers/thorchainSavers/utils', () => {
+  describe('getThorchainSaversQuote', () => {
+    it('gets a quote for a valid pool AssetId', async () => {
+      mockAxios.get.mockImplementationOnce(() =>
+        Promise.resolve({
+          data: btcQuote,
+        }),
+      )
+
+      const btcAssetMock = { assetId: btcAssetId, precision: 8 } as unknown as Asset
```

Comment:
not a huge fan of casting in tests - we have `btcAssetId` and can do

```
const btcAsset = getAssetService().getAll()[btcAssetId]
```

Reply:
036ff4dfeb52aea9414df73adc799816b87465a6